web/src/app/monitors/monitors-list/monitors-list.component.ts

```diff
@@ -35,6 +38,10 @@ export class MonitorsListComponent implements OnInit {
         take(1)
```

Comment:
I don't think `take` is needed here because the resolve has a `take` and it is a `snapshot`

Reply:
Right